usaspending_api/etl/award_helpers.py

```diff
@@ -0,0 +1,80 @@
+from django.db import connection
+
+
+def update_awards():
+    """
+    Update award data based on the information in their child transactions.
+
+    Note: Obviously the raw SQL is not ideal. That said, the complex update
```

Comment:
If we're certain this can't be done cleanly via the ORM; it would be useful to have a docstring explaining exactly what the SQL is doing so that it is more maintainable in the future for people less familiar with SQL.